doc/proposals/openshift-4.3/operator-testing-tool.md

```diff
@@ -0,0 +1,240 @@
+---
+title: operator-testing-tool
+authors:
+  - "@jmccormick2001"
+  - "@joelanford"
+reviewers:
+  - TBD
+  - "@joelanford"
+approvers:
+  - TBD
+creation-date: 2019-09-13
+last-updated: 2019-09-13
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# operator-testing-tool
+
+
+## Release Signoff Checklist
+
+- \[ \] Enhancement is `implementable`
+- \[ \] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Graduation criteria for dev preview, tech preview, GA
+- \[ \] User-facing documentation is created.
+
```

Comment:
It shows not be part of the template as well. IMHO: We should follow the template and any new good idea should be applied in the template before starting to be used. WDYT?